gradle/verification-metadata.xml

```diff
@@ -4107,6 +4112,11 @@
             <sha256 value="c43ecf17b539c777e15da7b5b86553b377e2d39a683de6285567d5283888e7ef" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.ow2.asm" name="asm-tree" version="9.7">
```

Comment:
we should probably remove the previous entries up to this version

Reply:
Older asm versions are still used by other modules (eg painless is on an older asm). We should look at cleaning up verification metadata separately, I think it is a gradle/build concern that needs to be handled consistently.

Reply:
I opened https://github.com/elastic/elasticsearch/issues/108825